src/Gateways/AbstractGateway.php

```diff
@@ -381,6 +402,31 @@ public function init_form_fields() {
 						'maxlength' => 18,
 					),
 				),
+			    'check_avs_cvv' => array(
+			        'title'				=> __( 'Check AVS CVN', 'globalpayments-gateway-provider-for-woocommerce' ),
+			        'label'				=> __( 'Check AVS/CVN result codes and reverse transaction.', 'globalpayments-gateway-provider-for-woocommerce' ),
+			        'type'				=> 'checkbox',
+			        'description'		=> sprintf(
+			            __( 'This will check AVS/CVN result codes and reverse transaction.' )
+			            ),
+			        'default'			=> 'no'
```

Comment:
We'll need to have this functionality enabled by default
```suggestion
			        'default'			=> 'yes'
```

Reply:
updated the code..